packages/typescript-estree/src/create-program/createProjectProgram.ts

```diff
@@ -35,7 +35,7 @@ function createProjectProgram(
     const errorLines = [
       '"parserOptions.project" has been set for @typescript-eslint/parser.',
       `The file does not match your project config: ${path.relative(
-        process.cwd(),
+        extra.tsconfigRootDir || process.cwd(),
```

Comment:
codecov complains due to this line, 
`tsconfigRootDir` is always set if user provides it or not, with exception to empty string